test/e2e/provider_expectations.go

```diff
@@ -140,7 +140,7 @@ func (tc *testConfig) ExpectNewNodeWhenDeletingMachine() error {
 	var triagedWorkerNode corev1.Node
 MachineLoop:
 	for _, m := range machineList.Items {
-		if m.Labels["sigs.k8s.io/cluster-api-machine-role"] == "worker" {
+		if m.Labels["machine.openshift.io/cluster-api-machine-role"] == "worker" || m.Labels["sigs.k8s.io/cluster-api-machine-role"] == "worker" {
```

Comment:
I won't block this PR on this, but we should really be using the already defined constants here.